vnext/src/Libraries/Network/RCTNetworkingWinShared.js

```diff
@@ -19,7 +19,14 @@ import type {RequestBody} from './convertRequestBody';
 
 class RCTNetworking extends NativeEventEmitter {
   constructor() {
-    super(RCTNetworkingNative);
+    const disableCallsIntoModule =
+      typeof global.__disableRCTNetworkingExtraneousModuleCalls === 'function'
+        ? global.__disableRCTNetworkingExtraneousModuleCalls()
+        : false;
+
+    super(RCTNetworkingNative, {
+      __SECRET_DISABLE_CALLS_INTO_MODULE_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: disableCallsIntoModule,
```

Comment:
>OR_YOU_WILL_BE_FIRE [](start = 52, length = 19)

This feels a little petty and unprofessional, not sure this should pass code review.

Reply:
IF this name is used in RN, we should add a comment stating that this name was out of our control, and perhaps file a bug on RN.

---
In reply to: [527037369](https://github.com/microsoft/react-native-windows/pull/6572#discussion_r527037369) [](ancestors = 527037369)

Reply:
This is a Facebook internal joke. They use the string in a whole bunch of places, which is how it got here.

Reply:
Here's a thread with some context, and some links to FB engineers talking about the string publicly. They use it in multiple codebases for bits that are internal and shouldn't be used. https://news.ycombinator.com/item?id=11447020

Reply:
You'd think it is okay to a comment referring to this?
```suggestion
      // This property name is part of the react-native library and not under react-native-windows control: See https://news.ycombinator.com/item?id=11447020
      __SECRET_DISABLE_CALLS_INTO_MODULE_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: disableCallsIntoModule,
```

Reply:
Some of the tooling we have looks for presence of diffs, so I'm not sure it's worth breaking that with some extra forking. This string is present in a decent number of places, and I can't imagine us changing FB code every time it shows up.

FWIW, I had a fun laugh at this string, but I also remember when Word engineers were upset because we had to take out all the swear words from out codebase 😀.

Reply:
Okay, sounds like a reasonable justification. 
It just feels unprofessional, my code ended has up as a post on the DailyWTF many years ago because of a function name and the embarrassment / unforfessionalism has stayed with me... one can still find some references